proxy/http/remap/RemapPlugins.cc

```diff
@@ -84,7 +84,7 @@ RemapPlugins::run_single_remap()
   url_mapping *map             = _s->url_map.getMapping();
   remap_plugin_info *plugin    = map->get_plugin(_cur); // get the nth plugin in our list of plugins
   TSRemapStatus plugin_retcode = TSREMAP_NO_REMAP;
-
+  int zret                     = 1;
```

Comment:
This, and the subsequent restructuring to have a single `return`, is to avoid duplicating the call to do the base rule remap, which was a comment on the original PR. Another alternative would have been to put that in a local lambda and call it where needed.

Reply:
It is going to change the order we see the `url_rewrite` debug statements appear, but it is probably fine.

Reply:
It's readable enough as-is. My comment was only meant as an observation. I think it's fine.